src/effects/native/balanceeffect.cpp

```diff
@@ -0,0 +1,176 @@
+#include "balanceeffect.h"
+
+#include "util/defs.h"
+
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
Could you explain why that would remove the ability to set the bypass knob fully dry?